src/components/hooks/usePerson.js

```diff
@@ -14,7 +14,12 @@ export default function usePerson(personID) {
 
   useEffect(() => {
     const onError = (error) => {
-      throw error;
+      setPerson({
+        displayName: ' ',
```

Comment:
What happens if you set displayName to `null` or `undefined`?

Reply:
nothing gets rendered

Reply:
You should have set this to empty string `''`, not space `' '`